Agents/CEAAgent/README.md

```diff
@@ -16,19 +16,19 @@ You will need to provide your credentials (github username/personal access token
     repo_username.txt
     repo_password.txt
 ```
-### Stack Set Up
+### 2.2. Stack Set Up
 The agent is designed to run in the stack. To start the stack, spin up the [Stack Manager](https://github.com/cambridge-cares/TheWorldAvatar/blob/main/Deploy/stacks/dynamic/stack-manager).
 
-### Access Agent Set Up
-The agent is designed to use the JPS Access Agent (```../AccessAgent```). Spin the access agent up as part of the same stack as spun up by the Stack Manager.
+### 2.3. Access Agent Set Up
+The agent is designed to use the [Access Agent](```../AccessAgent```). Spin the access agent up as part of the same stack as spun up by the Stack Manager.
```

Comment:
Sorry, you'll need to remove the triple quotes in order for the link to work correctly.